vnext/Desktop.DLL/React.Windows.Desktop.DLL.vcxproj

```diff
@@ -49,7 +49,6 @@
         REACTWINDOWS_BUILD - building with REACTWINDOWS_API as dll exports
       -->
       <PreprocessorDefinitions>REACTWINDOWS_BUILD;BOOST_ASIO_HAS_IOCP;_WINSOCK_DEPRECATED_NO_WARNINGS;_WIN32_WINNT=_WIN32_WINNT_WIN7;WIN32;_WINDOWS;_USRDLL;REACTNATIVEWIN32_EXPORTS;FOLLY_NO_CONFIG;NOMINMAX;GLOG_NO_ABBREVIATED_SEVERITIES;_HAS_AUTO_PTR_ETC;CHAKRACORE;RN_EXPORT=;%(PreprocessorDefinitions)</PreprocessorDefinitions>
-      <DebugInformationFormat>ProgramDatabase</DebugInformationFormat>
```

Comment:
>ProgramDatabase [](start = 30, length = 15)

Why is this removed?

Reply:
This was moved to property sheets as suggested by Julio.

---
In reply to: [300191022](https://github.com/microsoft/react-native-windows/pull/2677#discussion_r300191022) [](ancestors = 300191022)